### DISCUSSION TITLE
Docker instructions update

### DISCUSSION THREAD

**Discussion by olekhov** [smallstep/certificates#810] [General] [answered]
Hello.
Docker instructions on https://hub.docker.com/r/smallstep/step-ca need some update.
This line:
```
echo "<your password here>" > /home/step/secrets/password
```
Should be updated as:
```
echo -n "<your password here>" > /home/step/secrets/password
```
And I think you need explicitly state that `secrets/password` file SHOULD NOT contain new-line character (if some user wants to edit this file by her favorite editor by hand).

**Comment by tashian** [comment, marked as answer]
Hi, I've just tested this with step-ca 0.18.0 and it accepts a `--password-file` that contains one or more newlines at the end.

Can you share more about the operating environment where it's failing?

**Comment by olekhov** [comment]
Hello, seems to be duplicate of https://github.com/smallstep/cli/issues/428

When I tried to follow instructions, on step "4 Start step-ca" the docker just silently exits, no container was running.

I tried to investigate, removed `-d` flag (daemonize) and ended up with messages like these:
```
error decrypting /home/step/secrets/intermediate_ca_key: x509: decryption password incorrect
```
I searched for these and one of the answers suggested removing trailing `\n`. I tried from scratch and it worked.
I can not reproduce it now. 
Let's assume I have mistyped password at first attempt.
